RecoCTPPS/Configuration/python/recoCTPPS_DD_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.VeryForwardGeometry.geometryRPFromDD_2018_cfi import *
+#from Geometry.VeryForwardGeometry.geometryRPFromDD_2017_cfi import *
```

Comment:
Would you continue supporting the xml file access to geometry, shouldn't you rather choose  the specific year era based, instead of having to edit manually this file in the release?

Reply:
We decided that ERAs had a too large granularity (year) than what we expected to use. In fact, we may probebly have to chenge detectors during the year. So we decided to go with the Data Base. For the moment being we keep an xml support just for backup. We'll probably discard it once the DB is fully running and tested.